MonitorControl.xcodeproj/project.pbxproj

```diff
@@ -363,13 +363,13 @@
 				TargetAttributes = {
 					56754EAA1D9A4016007BCDC5 = {
 						CreatedOnToolsVersion = 8.0;
-						DevelopmentTeam = CYC8C8R4K9;
+						DevelopmentTeam = CUWL8LY6HS;
```

Comment:
Please also revert all changes in this file, especially the bundle identifier change